engine/src/main/java/org/camunda/bpm/engine/impl/persistence/entity/TimerEntity.java

```diff
@@ -100,6 +110,59 @@ protected void preExecute(CommandContext commandContext) {
     }
   }
 
+  protected String parseExpression(CommandContext commandContext) {
+    String expressionValue = null;
+    String expression = jobDefinition.getJobConfiguration().substring(7);
```

Comment:
❓ Should we add a comment for the "magic" `7` here? If we look at this in a while, we probably won't know why we use `7`. We could also create constants (here or somewhere else) that we base this number on (e.g. using the length of a String constant).